collections/Bodl/MS_Bodl_738.xml

```diff
@@ -30,30 +30,97 @@
                   </decoDesc>
+                  <additions>
+                     <p>The small, basically Anglicana glossing script, which rather erroneously have been attributed by Steele to Grandisson, are the second set of annotations in the copy, written later than those that are certainly Grandisson’s. They point to different emphases in commentary, such as <q>parabola</q> (the outer margin of fol. 27r), <q>historia</q> (the outer margins of fols 37r and 37v); spiritual topics (the outer margins of fols 1v, 101r, 155v, 171v, and 182r; the inner margin of fol. 197r, the outer margin of fol. 246r), natural phenomena (the outer margin of fols 45r, 179r–v, 180v, and 248v); definitions of figures of speech (the inner margin of fol. 66v); or names (<q>Rabimosse</q>, the outer margin of fol. 177v).</p>
+                  </additions>
                   <bindingDesc>
```

Comment:
I found this (additions) a bit unclear, as the information about Grandisson's glosses comes later in the description, and it might be clearer to say something like "two layers of laters glosses (1) by Grandission (for which see below, provenance) (2) by a later reader"